src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationapi/service/ClientsInterfaceService.kt

```diff
@@ -148,18 +145,19 @@ class ClientsInterfaceService(
     }
 
     val client = clientsByBaseClientId.last()
-    val registeredClient = registeredClientRepository.findByClientId(client.clientId)
-    val registeredClientBuilder = RegisteredClient.from(registeredClient)
+
+    val newClient = clientRepository.findClientByClientId(client.clientId)
```

Comment:
Do we need this look up - couldn't you just call copy on the client you already have?

Reply:
removed additional DB call